source/extensions/filters/http/ext_proc/processor_state.cc

```diff
@@ -440,6 +420,23 @@ void DecodingProcessorState::clearWatermark() {
   }
 }
 
+void DecodingProcessorState::clearRouteCache(const CommonResponse& common_response) {
+  if (!common_response.clear_route_cache()) {
+    return;
+  }
+  // Only clear the route cache if there is a mutation to the header and clearing is allowed.
+  if (filter_.config().disableClearRouteCache()) {
+    filter_.stats().clear_route_cache_disabled_.inc();
```

Comment:
Could you add back this `ENVOY_LOG(debug, "NOT clearing route cache, it is disabled in the config");`? 

This is meaningful here as it represents the case that ext_proc override the clear setting in common_response and force the clear off.

Reply:
Once it is configured, then this is in normal path, right? So, we will have such log for each response have this flag set.

Reply:
Let me add it back any way.

Reply:
The key we originally have log here is because of the override behavior that ext_proc/envoy overrides the ext_proc server's clear setting. This helps with observability/debuggability so that customer can figure out why their clear setting doesn't take effect. Same reason why we have stat here